### DISCUSSION TITLE
[Cool Idea]connect cloud storages to daedalOS

### DISCUSSION THREAD

**Discussion by 54145a** [DustinBrett/daedalOS#455] [Ideas]
Via [Alist](https://github.com/alist-org/alist) for instance

**Comment by DustinBrett** [comment]
Thanks for the link. Indeed that is a good idea. I'd like to start with OneDrive most likely because that is also actually integrated with Windows 10. The main reason I haven't gone with these approaches is that my app is client side only and I haven't desired to setup my own API keys to work with these applications.

I'd be interested if any of these methods Alist mentions can work without a server or API key of any kind, if so I would integrate them sooner.
